can/notifier.py

```diff
@@ -51,31 +147,36 @@ def __init__(
         #: Exception raised in thread
         self.exception: Optional[Exception] = None
 
-        self._running = True
+        self._stopped = False
         self._lock = threading.Lock()
 
         self._readers: List[Union[int, threading.Thread]] = []
-        buses = self.bus if isinstance(self.bus, list) else [self.bus]
-        for each_bus in buses:
+        self._bus_list = self.bus if isinstance(self.bus, list) else [self.bus]
```

Comment:
`self._bus_list` won't be correct after a user adds an additional Bus via `add_bus`, instead prefer to create an empty bus list here in the initializer, but add them inside the `add_bus` method.

Reply:
Good catch. The same was true for `self.bus`, i changed that to a property.